core/examples/driver/main.cpp

```diff
@@ -75,12 +75,20 @@ int main(int /*argc*/, char** /*argv*/)
 {
   char cmd[256];
 
-  std::vector<std::string> availableModules = GetExampleModules();
+  FrameworkFactory factory;
+  Framework* framework = factory.newFramework(std::map<std::string, std::string>());
```

Comment:
There should be an overload in the FrameworkFactory class of the `newFramework` method taking no arguments. See my comments for this class.